src/masterlist.py

```diff
@@ -91,94 +84,50 @@
     "Kuvaisit digiä!",
     "Ammattilainen ei napsi eikä räpsi. Hän hakee kuvat.",
     "Kissanpentuja kuvatessa väsytä ne leikkimällä ensiksi. Sen jälkeen ne jopa saattaa pysyä paikoillaan sen hetken.",
-    "Muista suojaviiva",
-    "Tarkista valkotasapaino ennen kun kuvaat",
+    "Tarkista valkotasapaino ennen kun kuvaat videota",
     "Fix it in post",
     "Suljinaika = 1/(2*framerate)",
-    "Muista purku",
     "Varmista että kuvakoko vaihtuu tai kuvakulma muuttuu riittävästi, jos leikkaat saman kohteen kuvien välillä",
-    "Staattinen kuva on joskus parempi kuin tärisevä ajo",
-    "Skarppaa!",
     "Käynnistä REC vasta kun klaffi on kuvassa. Tällöin se näkyy videon previkassa suoraan.",
-    "\"Slow is smooth and smooth is king\" -Stefan Czech",
     "Akkujen ja muistikorttien kanssa +1 -periaate, mukaan yksi enemmän kuin luulet tarvitsevasi",
-    "\"I have too much footage\" -No filmmaker ever",
-    "Oikee meininki on alkaa näpyttelee numeroo ja jos ei keksi mitään perustetta sinä aikana perua, painaa soittonappia.",
-    "Ei tommosia jäädä miettiin vaan ostetaan pois ja kattellaan sit et mitä tuli tehtyä.",
-    "Valokuvaus on vaa yks iso f-luvun maksimointitehtävä.",
-    "Jokaisessa digipokkarissa taitaa olla jonkinlainen zoom. Jos ei ole niin jätä kamera kauppaan.",
-    "Ledikeppiä voi himmentää sukkahousuilla.",
+    "Valokuvaus on yksi iso f-luvun maksimointitehtävä.",
     "Lightroomin print-osasto on ihan näppärä jos haluu esim järkkärikuvistaan pdf:ää. Tosta vaan valitsee kuvat jotka haluu samaan tiedostoon ja painaa print.",
-    "Osta mahdollisimman kallis kamera käytä siinä vain automaatti toimintaa.",
     "Kinohommissa kyllä todennu että se fiilis kuvatessa on kaikista tärkein. Jälki ei oo kuitenkaan hyvää niin siinä ei kannata sen kummemmin hifistellä.",
-    "Kunhan kameralla on kiva kuvata niin se on kivaa",
     "Kameratorilla siihen tuotteeseen laitetaan arvoa testaamalla ja parhaassa tapauksessa huoltamalla se, Rahalassa arvo tulee siitä että se on Rahala.",
-    "Amatöörit harrastkoot valokuvausta rauhassa kaduilla ja kerhoissaan.",
-    "Kivempi lukee tiliotetta, ku ostokset aina luottokorttilaskulla.",
-    "Ite otan yleensä lasin käteen niin, että vasen peukalo ja oikea keskisormi, joka painaa vapautuspainiketta, kohtaa, lasi menee paikoilleen. Voi vaihtaa pimeessäkin. Yleensä noissa on jotain tunnettavaa, joilla ton kohdan tunnistaa kokeilemalla. Korttipesän valo on pimeen puuhailuihin omasta mielestä parasta.",
     "Syväterävyysalue on verrannollinen etäisyyden neliöön, ja kääntäen verrannollinen polttovälin neliöön.",
     "Tarkennus on on aina kuvaajan tietoinen valinta.",
     "Treenaa forkkuja.",
```

Comment:
Tää pois. Turha ja tylsä imo.

Reply:
En ollu varma mitä meinasit, mutta tein jotain. Forkkutipsi on erittäin hyvä vaikka itse sanonkin.